bindings/mqtt/0.2.0/server.json

```
@@ -1,23 +1,24 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "http://asyncapi.com/bindings/mqtt/0.2.0/server.json",
-  "title": "MQTT server bindings object",
+  "$id": "http://asyncapi.com/bindings/mqtt/0.0.2/server.json",
+  "title": "Server Schema",
   "description": "This object contains information about the server representation in MQTT.",
   "type": "object",
   "additionalProperties": false,
   "patternProperties": {
-    "^x-[\\w\\d\\.\\x2d_]+$": {
+    "^x-[\\w\\d\\.\\-\\_]+$": {
```

That's an invalid pattern. Should stay as it is.
```suggestion
    "^x-[\\w\\d\\.\\x2d_]+$": {
```